services/src/main/java/org/exoplatform/webconferencing/WebConferencingService.java

```diff
@@ -2342,6 +2352,7 @@ private void shareRecordToUser(String user, Node recordNode, String perm) {
       String nodeMimeType = org.exoplatform.wcm.ext.component.activity.listener.Utils.getMimeType(recordNode);
       link.addMixin(NodetypeConstant.MIX_FILE_TYPE);
       link.setProperty(NodetypeConstant.EXO_FILE_TYPE, nodeMimeType);
+      link.setProperty(EXO_TITLE_PROP, recordNode.getName());
```

Comment:
Please check if added/add the mixin **EXO_RSS_ENABLE_PROP** before addding the property **EXO_TITLE_PROP**